docs/guide/convert_doc.md

```diff
@@ -103,3 +103,55 @@ Simply pass a python list object containing multiple urls. Let `URL` be a list c
     # Let's download all the converted documents locally in RESULT_DIR
     documents.download_all(result_dir = RESULT_DIR)
     ```
+
+---
+
+## Downloading converted documents
```

Comment:
What is actually different in this section and examples, compared to the previous one? It looks exactly the same code to me.